pkg/controller/container-runtime-config/helpers.go

```diff
@@ -392,6 +367,52 @@ func updateRegistriesConfig(data []byte, internalInsecure, internalBlocked []str
 	return newData.Bytes(), nil
 }
 
+// updatePolicyJSON decodes the data rendered from the template, merges the changes in and encodes it
+// back into a JSON format. It returns the bytes of the encoded data
+// It also returns an error if both allowed and blocked registries are set
+func updatePolicyJSON(data []byte, internalBlocked, internalAllowed []string) ([]byte, error) {
```

Comment:
Please add something like
> WARNING: This can not safely edit policy files with arbitrary complexity, especially files which include `signedBy` requirements. It expects the input policy to be generated by templates in this project.

Reply:
Fixed